tests/schema-sql.test.mjs

```
@@ -0,0 +1,133 @@
+register()
+
+const src = name => pathToFileURL(join(repo, 'src/core/storage/message-store', name)).href
+const { SQL } = await import(src('sql.ts'))
+const { createSchema, createUploadCacheSchema, dropLegacySchema } = await import(src('schema.ts'))
+const { hashId } = await import(src('hash.ts'))
+
+const file = join(mkdtempSync(join(tmpdir(), 'qqbot-test-')), 'test.db')
+const db = await new Promise((resolve, reject) => {
+  const d = new sqlite3.Database(file, err => err ? reject(err) : resolve(d))
+})
+const run = (sql, params = []) => new Promise((res, rej) => db.run(sql, params, e => e ? rej(new Error(`${e.message}\nSQL: ${sql}`)) : res()))
```

ESLint

[error] Promise constructor parameters must be named to match "^_?resolve$"

(promise/param-names)